### ISSUE TITLE
[EP] EVA 3 E3D Revo V6 Support

### ISSUE THREAD

**Issue by githubcura35** [EVA-3D/eva-main#87]
EVA 3 now supports the E3D Revo Voron. Are there any plans to support the E3D Revo Six? (https://e3d-online.com/products/revo-six) Any plans to support V6 based hotends at all?

**Comment by pkucmus** [comment]
This would also mean support for V6, right? Not sure about the timing but something I'd like to address

**Comment by huzargarden** [comment]
Definitely, something that I'll be looking for. I like the EVA design and I'm very happy with my V6 Hot End.

**Comment by a937983423** [comment]
I would like to ask why the most popular E3D_ V6 does not support

**Comment by pkucmus** [comment]
> I would like to ask why the most popular E3D_ V6 does not support

According to the EVA survey taken more than a year ago, V6 had 11.5% of the usage I can only imagine that number being lower now. It's an old and relatively underperforming hotend, it's popularity overall means nothing in terms of popularity in the scope of EVA users. Also you asked the question in the ticket that is here to add support for that hotend so in case you just wanted to see an answer it's as above, otherwise, it's just patience.

**Comment by fj-guy** [comment]
The old e3d v6 served me well for quite awhile.  Since the Revo 6 was a direct drop-in replacement I switched to that. And I would imaginge a lot of V6 users did the same.  Not all of us care to print at 300mm/s with crazy accel values.  I recently upgraded to the V-CORE-3.1 but had retained the older 2.4 carriage because I didn't want to upgrade to a HF hotend just yet.  Luckily someone did a V6 mount for EVA 3.  Do your survey on REVO 6 users instead of V6 users.

**Comment by MechanizedMedic** [comment]
I just got done drawing up a [to-spec remix](https://cad.onshape.com/documents/aa6b87b04c53d0436ebc5b95/w/52b723d56fd8bb8eb734a789/e/abb8bc791c22639c7a986955?configuration=List_G0JwUrcDLEaXCX%3DBL_Touch%3BList_Ojsu1VJQPwv42D%3DBMG%3BList_S9vpzEZQSzFGKh%3DVolcano%3BList_SYMLdq345kAX9B%3DVolcano%3BList_durAPyfJdcZirJ%3D_5015%3BList_vN2ntClvCm9on0%3DNone&renderMode=0&rightPanel=configPanel&uiState=63578f586b23cf60326f8b8e) that is compatible with Volcano/V6 (if push fitting is removed). It should allow the Volcano and v6 to use the same hotend block with the standard cooling duct.

I also made a cable mount for the BMG extruder but it is only compatible with the single 50mm blower for now.

Feedback and collaboration are welcome! :)

**Comment by githubcura35** [comment]
@MechanizedMedic Thanks a lot. I am unable to export to STL. Can you grand permissions in Onshape?

**Comment by MechanizedMedic** [comment]
@githubcura35, yep sorry about that. Should be good now.

**Comment by githubcura35** [comment]
@MechanizedMedic Thanks a lot. I will check it out but feedback will take some time.

**Comment by githubcura35** [comment]
@MechanizedMedic The V6 fits perfectly but I am having some trouble with the cooling. The duct from the default bottom reaches furhter down than the tip of the nozzle
![eva3-v6](https://user-images.githubusercontent.com/43914330/198972685-402c1c8e-48b3-4855-9759-e69e2f21cde3.jpg)

**Comment by MechanizedMedic** [comment]
@githubcura35 It should line up if you move the "hotend_volcano" block down one slot in the "universal_front" reciever. I tried to size it so the Volcano or V6 would fit by moving this mounting location 

![Like this.](https://user-images.githubusercontent.com/9471777/199035307-367e19fc-4088-408f-ba20-01d8cd425008.png)

**Comment by githubcura35** [comment]
@MechanizedMedic There is no way to move "hotend_volcano" further down as the zip tie holder collides with the "universal_front".
![colision](https://user-images.githubusercontent.com/43914330/199229292-1c22e319-ae26-450e-bfbe-cea30c84e73c.PNG)

To make that work the zip tie holder needs to be moved further up in the "hotend_volcano" or we need a new "v6_front".
As a workaround I could grind of a little bit from the "universal_front".

**Comment by MechanizedMedic** [comment]
Okay, thanks for testing. I'll fix it up later this evening and repost.

**Comment by olivbd** [comment]
Why not use this Printables model? https://www.printables.com/model/201093-eva-30-e3dv6-revo-revo-micro-hotends-mount

I printed and used it for several months without any problem...

**Comment by MechanizedMedic** [comment]
@githubcura35, I updated the design this morning to move the hotend down 7.5mm. This should put the v6 in the correct spot for the standard duct. The Volcano hotend will now need the Volcano duct.

**Comment by githubcura35** [comment]
@MechanizedMedic Thanks a lot for your effort. The "hotend_volcano" now perfectly fits the "universal_front". I have not done any actual printing as my custom printer is currently not ready for printing. This will take some time but from what I can tell this looks very promissing.

**Comment by TekuSP** [comment]
> Why not use this Printables model? https://www.printables.com/model/201093-eva-30-e3dv6-revo-revo-micro-hotends-mount
> 
> I printed and used it for several months without any problem...

That is REVO V6 and not V6 standard

**Comment by olivbd** [comment]
> > Why not use this Printables model? https://www.printables.com/model/201093-eva-30-e3dv6-revo-revo-micro-hotends-mount
> > I printed and used it for several months without any problem...
> 
> That is REVO V6 and not V6 standard

No, there are two versions: Revo V6 and V6 standard.
